src/main/java/apoc/schema/Schemas.java

```diff
@@ -170,7 +283,7 @@ private AssertSchemaResult createSinglePropertyIndex(Schema schema, String lbl,
 
     private AssertSchemaResult createCompoundIndex(String label, List<String> keys) {
         List<String> backTickedKeys = new ArrayList<>();
-        keys.forEach(key->backTickedKeys.add(String.format("`%s`", key)));
+        keys.forEach(key -> backTickedKeys.add(String.format("`%s`", key)));
```

Comment:
should be just Util.quote

Reply:
We didn't change anything in this function